python/ray/dataframe/test/test_dataframe.py

```diff
@@ -2150,7 +2168,7 @@ def test_reindex():
     ray_df = create_test_dataframe()
 
     with pytest.raises(NotImplementedError):
-        ray_df.reindex()
+        ray_df.reindex(None)
```

Comment:
Please add a sanity check test here.

Reply:
Looking through the code, was reindex dropped from this PR?

Reply:
yes, reindex did not have full functionality (axes, errors) so I took it out for the PR against master